src/output/icons.rs

```diff
@@ -81,12 +85,17 @@ lazy_static! {
         m.insert("hidden", '\u{f023}'); // 
         m.insert("include", '\u{e5fc}'); // 
         m.insert("lib", '\u{f121}'); // 
+        m.insert("LICENSE", '\u{f718}'); // 
```

Comment:
This symbol doesn't render for me

Reply:
same document icon 'nf-md-file_document'